package.json

```diff
@@ -1266,6 +1266,19 @@
                     "description": "Whether to use OAuth credential for Cosmos DB NoSQL resources",
                     "default": false
                 },
+                "azureDatabases.confirmationStyle": {
+                    "type": "string",
+                    "description": "The behavior to use when confirming operations that cannot be undone, such as deleting resources.",
+                    "enum": [
```

Comment:
The enum values should be unique and internal for localisation purposes, such as `WordConfigmration` and `ButtonClick`, otherwise localisation will be difficult. There is an additional `enumItemLabels` property to define labels for the values, which can be localised afterwards. See the docs: https://code.visualstudio.com/api/references/contribution-points#Configuration-property-schema

Reply:
Thanks, I started using `enumItemLabels`